src/routes/tasks.rs

```diff
@@ -102,6 +103,8 @@ pub mod collector_auth_tokens {
         let leader = task.leader_aggregator(&db).await?;
         let client = leader.client(client);
         let task_response = client.get_task(&task.id).await?;
-        Ok(Json([task_response.collector_auth_token]))
+        Ok(Json([task_response
```

Comment:
I'm surprised this is what rustfmt likes but who am I to second-guess the linter